serving/docker/fastertransformer.Dockerfile

```diff
@@ -67,19 +68,14 @@ RUN apt-get update && apt-get install -y wget git zlib1g-dev && \
 
 ENV PATH=/usr/local/mpi/bin:${PATH} LD_LIBRARY_PATH=/usr/local/mpi/lib:${LD_LIBRARY_PATH}
 
-# Supporting build g4,g5,p3,p4
-RUN git clone https://github.com/NVIDIA/FasterTransformer.git -b ${ft_version} \
-    && mkdir -p FasterTransformer/build \
-    && cd FasterTransformer/build \
-    && git submodule init && git submodule update \
-    && cmake -DCMAKE_BUILD_TYPE=Release -DSM=70,75,80,86 -DBUILD_PYT=ON -DBUILD_MULTI_GPU=ON .. \
-    && make -j$(nproc) install \
-    && rm -rf lib/*TritonBackend.so \
-    && cp lib/*.so /usr/local/backends/fastertransformer/ \
-    && mkdir -p /usr/local/backends/fastertransformer/bin \
-    && cp -r bin/*_gemm /usr/local/backends/fastertransformer/bin/ \
-    && cp ../LICENSE /root/FASTERTRANSFORMER_LICENSE \
-    && cd ../../ && rm -rf FasterTransformer
+# Install fastertransformer and triton
+RUN mkdir -p /usr/local/backends/fastertransformer && mkdir -p /usr/local/tritonserver && \
+    curl -o /usr/local/tritonserver/libtritonserver.so https://publish.djl.ai/triton/${triton_version}/libtritonserver.so && \
```

Comment:
```suggestion
    curl -o /usr/local/tritonserver/libtritonserver.so https://publish.djl.ai/tritonserver/${triton_version}/libtritonserver.so && \
```

Reply:
You also need the following files:

- libdcgm.so.2
- libnuma.so.1
- libre2.so.5